Example/iOS/ViewControllers/AnimationPreviewViewController.swift

```diff
@@ -46,11 +46,19 @@ class AnimationPreviewViewController: UIViewController {
     animationView.backgroundBehavior = .pauseAndRestore
     animationView.translatesAutoresizingMaskIntoConstraints = false
 
+    engineLabel.font = .preferredFont(forTextStyle: .footnote)
```

Comment:
While I was here I also added a label to the example app's preview VC that tells you which rendering engine is being used